backend/src/routes/api/configmaps/index.ts

```diff
@@ -0,0 +1,58 @@
+import { KubeFastifyInstance } from '../../../types';
+import { FastifyReply, FastifyRequest } from 'fastify';
+import { V1ConfigMap } from '@kubernetes/client-node';
+
+module.exports = async (fastify: KubeFastifyInstance) => {
+  fastify.get('/:name', async (request: FastifyRequest<{ Params: { name: string } }>) => {
+    const cmName = request.params.name;
+    try {
+      const cmResponse = await fastify.kube.coreV1Api.readNamespacedConfigMap(
+        cmName,
+        fastify.kube.namespace,
+      );
+
+      return cmResponse.body;
+    } catch (e) {
+      if (e.response?.statusCode === 404) {
+        return null;
+      }
+      fastify.log.error(`Could not read config map ${cmName}, ${e}`);
+    }
```

Comment:
I'm not sure why we don't want to send back the 404? If the ConfigMap is not created in the desired namespace, it **should** be 404

@DaoDaoNoCode @maroroman Help me understand this (as I understanding it was a conversation you two had)